src/sentry/static/sentry/app/views/performance/table.tsx

```diff
@@ -211,23 +251,50 @@ class Table extends React.Component<Props, State> {
     this.setState({widths});
   };
 
+  getSortedEventView() {
+    const {eventView} = this.props;
+
+    // We special case sort by key transactions here to include
+    // the transaction name and project as the secondary sorts.
+    const keyTransactionSort = eventView.sorts.find(
+      sort => sort.field === 'key_transaction'
+    );
+    if (keyTransactionSort) {
+      const allowedFields = eventView.getFields();
+      const sorts = ['key_transaction', 'transaction', 'project']
+        .filter(field => allowedFields.includes(field))
```

Comment:
Do you need this step? Isn't the field list for performance table fixed?

Reply:
It was just to be on the safe side, but I can remove it.